packages/webapp/pages/notifications.tsx

```diff
@@ -50,12 +56,31 @@ const Notifications = (): ReactElement => {
         if (hasUnread(data)) {
           readNotifications();
         }
+
+        if (queryResult.isFetchedAfterMount && !hasTrackedFetch) {
+          trackEvent({ event_name: AnalyticsEvent.OpenNotificationList });
+          setHasTrackedFetch(true);
+        }
       },
     },
   );
 
   const length = queryResult?.data?.pages?.length ?? 0;
 
+  const onNotificationClick = (type: NotificationType) => {
+    trackEvent({
+      event_name: AnalyticsEvent.ClickNotification,
+      extra: JSON.stringify({ origin: Origin.NonRealTime, type }),
+    });
+  };
+
+  useEffect(() => {
+    trackEvent({
+      event_name: AnalyticsEvent.Impression,
+      extra: JSON.stringify({ notifications_number: unreadCount }),
+    });
+  }, []);
```

Comment:
This should be tracked whenever we show the bell icon with at least one unread count, not here

Reply:
Oh, cool. I misunderstood it for if the icon is filled which only happens here. Thanks for clearing it up.